cpp/tests/utilities/column_utilities.hpp

```diff
@@ -94,7 +94,7 @@ std::string to_string(cudf::column_view const& col, std::string const& delimiter
  *
  * @param col The column view
  *---------------------------------------------------------------------------**/
-std::vector<std::string> to_strings(cudf::column_view const& col);
+thrust::host_vector<std::string> to_strings(cudf::column_view const& col);
```

Comment:
This change seems pretty pervasive. I would prefer to stick with STL containers wherever possible.

Reply:
See comment above. But basically everywhere I use `thrust::host_vector` is necessary. I do agree that using STL containers as much as possible is preferable and that is what we are dong :)